### PR TITLE
feat: Improve dev workflow with delta, zsh fixes, and pr-review restoration

### PR DESCRIPTION
**Pull request by evansenter**
## Summary

- Add delta as git pager with Catppuccin Mocha theme and side-by-side diffs
- Add git aliases: `file-diff`, `file-diffu`, `diffu` for viewing file history diffs
- Enable emacs keybindings in zsh (fixes Ctrl+A, Ctrl+E, Ctrl+W, etc.)
- Restore over-trimmed pr-review.md sections that were causing ambiguity
- Fix user_guide.md event bus examples with missing session_id

### Git config
- Delta pager with syntax highlighting, line numbers, side-by-side view
- `git file-diff N <file>` - combined diff of last N commits touching a file
- `git diffu` / `git file-diffu` - unified (non-side-by-side) versions for copying

### Zsh
- `bindkey -e` to enable emacs keybindings (was defaulting to vi mode)

### pr-review.md restoration
- Coverage severity classification (Important vs Suggestion)
- AskUserQuestion batching rules and Recommended alignment
- Defer priority label guidance
- Filter resolved items parsing instructions  
- Key Principle section

## Test plan

- [ ] Run `git diff HEAD~1` - should show delta side-by-side view
- [ ] Run `git file-diff 2 home/.gitconfig` - should show combined diff
- [ ] Open new terminal, verify Ctrl+A goes to start of line
- [ ] Review pr-review.md for completeness

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: home/.claude/commands/pr-review.md [hunk 1]
```diff
@@ -34,13 +34,20 @@ Skill(pr-review-toolkit:review-pr)
...
+- Find new/modified source files, check for corresponding tests
+- Flag: new public APIs without tests (Important), missing edge cases (Suggestion)
+
+**Example coverage:**
+- Identify user-facing features (new commands, flags, public APIs)
+- Flag: new user-facing feature without example (Important)
 
 ### 4. Run Examples (if applicable)
 
 For significant changes, run examples with debug logging (check CLAUDE.md for flags like `LOUD_WIRE=1`). Flag issues found.
```
Comment (evansenter) [inline home/.claude/commands/pr-review.md]: Log output to tmp/ and share with user for optional inspection.

File: home/.claude/commands/pr-review.md [hunk 2]
```diff
@@ -34,13 +34,20 @@ Skill(pr-review-toolkit:review-pr)
...
 
+**Skip if:** Documentation-only changes or no debug flags defined.
```
Comment (evansenter) [inline home/.claude/commands/pr-review.md]: **Skip if:** Documentation-only changes

File: docs/user_guide.md [hunk 3]
```diff
@@ -454,6 +455,7 @@ When you learn something non-obvious:
 mcp__event-bus__publish_event(
   event_type="gotcha_discovered",
   payload="SQLite datetime needs explicit adapters in Python 3.12+",
+  session_id: "<your-session-id>",
   channel="repo:claude-session-analytics"
```
Comment (evansenter) [inline docs/user_guide.md]: channel=“repo:my-app"

File: home/.claude/commands/pr-review.md
Comment (evansenter) [inline home/.claude/commands/pr-review.md]: is it clear that this should be posted to the PR?
